installer/lib/kitto_new.ex

```diff
@@ -48,6 +50,7 @@ defmodule Mix.Tasks.Kitto.New do
     {:text, "new/assets/stylesheets/application.scss", "assets/stylesheets/application.scss"},
     {:keep, "new/public/assets",                       "public/assets"},
     {:text, "new/public/assets/favicon.ico",           "public/assets/favicon.ico"},
+    {:text, "new/public/assets/images/nyan.gif",       "public/assets/images/nyan.gif"},
```

Comment:
Can you change it to a placeholder image?

![image_widget](https://cloud.githubusercontent.com/assets/645514/21074956/2a7dddfa-bf0f-11e6-9b71-776233735685.png)